### PR TITLE
feat: rebrand to testbeats

### PR DESCRIPTION
**Pull request by ASaiAnudeep**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (leelaprasadv) [inline .vscode/settings.json:1]: what does this do? It should have been filtered out in git ignore isn't it?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
 <span align="center">
 
-![logo](https://github.com/test-results-reporter/reporter/raw/main/assets/logo.png)
+![logo](https://github.com/test-results-reporter/testbeats/raw/main/assets/logo.png)
 
-## Test Results Reporter
```
Comment (leelaprasadv) [inline README.md:5]: Missed renaming in the Heading?

File: package.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "test-results-reporter",
-  "version": "1.1.6",
+  "name": "testbeats",
+  "version": "2.0.0",
```
Comment (leelaprasadv) [inline package.json:3]: If it's a new repo let's reset the version?

File: README.md (hunk 2 above)
Comment (leelaprasadv) [reply README.md:5]: Let's also include a reminder or a notification about the rebranding?

File: .vscode/settings.json (hunk 1 above)
Comment (ASaiAnudeep) [reply .vscode/settings.json:1]: this was intentional. I have code spell checker plugin.

File: package.json (hunk 3 above)
Comment (ASaiAnudeep) [reply package.json:3]: as we are using the same repo, we already have earlier versions registered in releases. so I bumped the major version.

**Review by leelaprasadv — APPROVED** [review]

**Pull request merged**